llvm/test/lab2/borovkov_sergey/test.ll

```diff
@@ -0,0 +1,149 @@
+; RUN: opt -load-pass-plugin=%llvmshlibdir/MulShiftsBorovkovPlugin%shlibext -passes=borovkovmulshifts -S %s | FileCheck %s
+
+;int f1(int a){
+;    int c = a + 4;
+;    return c;
+
+define dso_local i32 @f1(i32 noundef %0) #0 {
+  %2 = alloca i32, align 4
+  %3 = alloca i32, align 4
+  store i32 %0, ptr %2, align 4
+  %4 = load i32, ptr %2, align 4
+  %5 = add nsw i32 %4, 4
+  store i32 %5, ptr %3, align 4
+  %6 = load i32, ptr %3, align 4
+  ret i32 %6
+}
+
+; CHECK-LABEL: @f1
+; CHECK: %4 = load i32, ptr %2, align 4
+; CHECK-NEXT: %5 = add nsw i32 %4, 4
```

Comment:
It is better to avoid using of specific numbers in IR. Match them by regex
```suggestion
; CHECK: %[[load:[0-9]+]] = load i32, ptr %2, align 4
; CHECK-NEXT: %5 = add nsw i32 %[[load]], 4
```
and so on...

Reply:
Done.